packages/ember-simple-auth/.eslintignore

```diff
@@ -0,0 +1,18 @@
+/blueprints/*/files
+
+# unconventional js
+/blueprints/*/files/
+/vendor/
+
+# compiled output
+/dist/
+/tmp/
+
+# dependencies
+/bower_components/
+
+# misc
+/coverage/
+
+# ember-try
+/.node_modules.ember-try/
```

Comment:
what about `.bower_components.ember-try`? otherwise we probably don't need the `bower_components` above either, right?

Reply:
I removed `bower_components` above 👍